observability/WFLY-19846-promote-mp-tel-2-to-default.adoc

```diff
@@ -0,0 +1,115 @@
+---
+categories:
+- microprofile
+- observability
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/619
+feature-team:
+ developer: jasondlee
+ sme:
+  -
+ outside-perspective:
+  -
+promotes: https://issues.redhat.com/browse/WFLY-19590
+promoted-by: https://github.com/wildfly/wildfly-proposals/issues/619
+---
+= [Default] Promote MicroProfile Telemetry 2.0 to WildFly Standard
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+MicroProfile Telemetry 2.0 shipped with WildFly Preview 34. To meet WildFly and product roadmap goals, this feature needs to be migrated to the WildFly feature pack, and its stability level promoted to `DEFAULT`.
+
+=== User Stories
+
+User wants to deploy an application using MicroProfile Telemetry 2.0 from MicroProfile Platform 7.0.
```

Comment:
Can you please elaborate how users can benefit from new features brought by MP Telemetry 2.0? https://microprofile.io/specifications/telemetry/2-0/ describes new features against MP Telemetry 1.1 that we have earlier

Reply:
@jasondlee Thank you for the update. What about integration between Telemetry and Fault Tolerance? My understanding is that WF users can benefit from it as well?

I would like to explain my motivation in this discussion, because I understand that this could be considered as nit-picking. I'm not sure about original reason why "User stories" section has been added to the template for these analysis. But I can guess - I would say that identification of important user stories is useful for later parts of process, specifically for documentation. I really don't want to push on process/analysis without no reasons, but I actually believe that early identification of user-stories can improve documentation, which will be useful of users at the end.

Reply:
AFAIU server is going to implement micrometer metrics and telemetry metrics. Telemetry seems to be newer and also is a part of spec. Possible user story could be also to migrate micrimeter metrics to telemetry metrics?

Reply:
> @jasondlee Thank you for the update. What about integration between Telemetry and Fault Tolerance? My understanding is that WF users can benefit from it as well? cc @fabiobrz

MP Fault Tolerance is listed in the `Affected Projects or Components` section. I don't see the need to list out every feature of the technology.

Reply:
Thank you for your feedback! I tried to explain the motivation behind this above. If that's not a reason for you, we can consider this topic as resolved (in scope of analysis) and handle documentation separately in later RFE phases.

Reply:
> AFAIU server is going to implement micrometer metrics and telemetry metrics. Telemetry seems to be newer and also is a part of spec. Possible user story could be also to migrate micrimeter metrics to telemetry metrics?

I think migration only makes sense if we're encouraging or requiring users to move, which we are not at this point.

Reply:
+1, thank you for your feedback!